tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -207,49 +207,14 @@ def get_subtype_from_configdb(duthost):
     return subtype_exist, subtype_value
 
 
-@pytest.fixture(scope="module", params=[SINGLE_TOR_MODE, DUAL_TOR_MODE])
-def testing_config(request, duthosts, rand_one_dut_hostname, tbinfo):
-    testing_mode = request.param
+@pytest.fixture(scope="module")
+def testing_config(duthosts, rand_one_dut_hostname, tbinfo):
     duthost = duthosts[rand_one_dut_hostname]
-    subtype_exist, subtype_value = get_subtype_from_configdb(duthost)
 
     if 'dualtor' in tbinfo['topo']['name']:
-        if testing_mode == SINGLE_TOR_MODE:
-            pytest.skip("skip SINGLE_TOR_MODE tests on Dual ToR testbeds")
-
-        if testing_mode == DUAL_TOR_MODE:
-            if not subtype_exist or subtype_value != 'DualToR':
-                assert False, "Wrong DHCP setup on Dual ToR testbeds"
-
-            yield testing_mode, duthost, 'dual_testbed'
-    elif tbinfo['topo']['name'] in ('t0-54-po2vlan', 't0-56-po2vlan'):
-        if testing_mode == SINGLE_TOR_MODE:
-            if subtype_exist and subtype_value == 'DualToR':
-                assert False, "Wrong DHCP setup on po2vlan testbeds"
-
-            yield testing_mode, duthost, 'single_testbed'
-
-        if testing_mode == DUAL_TOR_MODE:
-            pytest.skip("skip DUAL_TOR_MODE tests on po2vlan testbeds")
+        yield DUAL_TOR_MODE, duthost
     else:
-        if testing_mode == DUAL_TOR_MODE:
-            pytest.skip("skip DUAL_TOR_MODE tests on Single ToR testbeds")
-
-        if testing_mode == SINGLE_TOR_MODE:
-            if subtype_exist:
-                duthost.shell('redis-cli -n 4 HDEL "DEVICE_METADATA|localhost" "subtype"')
-                restart_dhcp_service(duthost)
-
-        if testing_mode == DUAL_TOR_MODE:
-            if not subtype_exist or subtype_value != 'DualToR':
-                duthost.shell('redis-cli -n 4 HSET "DEVICE_METADATA|localhost" "subtype" "DualToR"')
```

Comment:
Can you confirm it is safe to remove this key setting? No one depends on/consumes this.

Reply:
dhcp_relay container would consume it, but seems this value should be parsed from minigraph by default if there is a PEER_SWITCH (DualToR scenario)
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-config-engine/minigraph.py#L2164

Reply:
ok, looks we are safe here